roles/cnf_cert/tasks/pre-run.yml

```diff
@@ -148,28 +148,28 @@
       # On RHEL8 git clone can sporadically fail with OpenSSL SSL_read:
       # SSL_ERROR_SYSCALL, errno 104. This is a workaround to try cloning the
       # repo multiple times.
-      register: test_network_function_gitref
+      register: redhat_best_practices_for_k8s_gitref
       retries: 3
       delay: 30
-      until: not test_network_function_gitref.failed
+      until: not redhat_best_practices_for_k8s_gitref.failed
 
     # For HEAD version, the support_image_version should be 'latest'. Else, check version.json file
     # from certsuite repo
     - name: Set support_image_version variable
       vars:
         _cc_version_json_file: "{{ lookup('file', tnf_dir + '/version.json') | from_json }}"
       ansible.builtin.set_fact:
-        support_image_version: "{{ 'latest' if test_network_function_version == 'HEAD' else _cc_version_json_file.partner_tag[1:] }}" # noqa: redhat-ci[no-role-prefix]
+        support_image_version: "{{ 'latest' if redhat_best_practices_for_k8s_version == 'HEAD' else _cc_version_json_file.partner_tag[1:] }}" # noqa: redhat-ci[no-role-prefix]
 
     # We need to pull the original image to tag it. Then, we remove the pulled
     # image
     - name: Pull and tag tnf image for the version under test
       ansible.builtin.shell: |
         set -ex
-        podman pull quay.io/testnetworkfunction/{{ test_network_function_project_name }}:{{ tnf_version_image }}
-        podman tag quay.io/testnetworkfunction/{{ test_network_function_project_name }}:{{ tnf_version_image }} \
+        podman pull quay.io/redhat-best-practices-for-k8s/{{ redhat_best_practices_for_k8s_project_name }}:{{ tnf_version_image }}
+        podman tag quay.io/redhat-best-practices-for-k8s/{{ redhat_best_practices_for_k8s_project_name }}:{{ tnf_version_image }} \
```

Comment:
For this role `cnf_cert` I believe is not needed to change the values either, because it needs to use the old versions, right. I don't think quay introduces a redirect as github when renaming registries. Correct me if I'm wrong but I believe quay is a new registry with new images.

```Shell
❯ skopeo inspect docker://quay.io/testnetworkfunction/cnf-certification-test:v5.2.2 | jq .Name
"quay.io/testnetworkfunction/cnf-certification-test"

dci-openshift-agent on  upstream_catalog [?] took 13s 


❯ skopeo inspect docker://quay.io/redhat-best-practices-for-k8s/certsuite:v5.2.2 | jq .Name
FATA[0000] Error parsing image name "docker://quay.io/redhat-best-practices-for-k8s/certsuite:v5.2.2": reading manifest v5.2.2 in quay.io/redhat-best-practices-for-k8s/certsuite: manifest unknown 
```

Reply:
Good, but it does not include `v5.2.2`, I assume that version will not be in the new repo, right?